R/plotting-functions.R

```diff
@@ -0,0 +1,223 @@
+#' @title Plotting Spectra
```

Comment:
Add a reference to this man page in the `Spectra` documentation.

Reply:
No problem adding a reference or adding you also as an author of the function - but to which function should I refer? The `MSnbase::plot`?

Reply:
I simply meant a sentence in the Spectra man page that says something along the lines of: _For details on plotting Spectra, see.... _

Reply:
Done (see commit below).

Reply:
Sorry, I wasn't being clear. Will do it myself.